common/test/acceptance/pages/studio/import_export.py

```diff
@@ -171,15 +171,17 @@ def _wait_for_button(self):
             timeout=30
         ).fulfill()
 
-    def  upload_tarball(self, tarball_filename):
+    def upload_tarball(self, tarball_filename):
         """
         Upload a tarball to be imported.
         """
         asset_file_path = self.file_path(tarball_filename)
         # Make the upload elements visible to the WebDriver.
         self.browser.execute_script('$(".file-name-block").show();$(".file-input").show()')
-        self.q(css='input[type="file"]')[0].send_keys(self.file_path('test.tar.gz'))
         self.q(css='input[type="file"]')[0].send_keys(asset_file_path)
+        self.q(css='input[type="file"]')[0].send_keys(asset_file_path)
```

Comment:
Just to be on the safe side of some unforeseeable problems, I uploaded the same file twice and removed hypothetical test.tar.gz file upload.